### PR TITLE
Update config.js:    tls.rejectUnauthorized false by default (for selfsigned certificates)

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -119,6 +119,12 @@ config.smtp = {
      pass: 'xyz'
      }
     */
+    ,
+    tls: {
```
Comment (fgalan) [inline config.js]: Should the new option be document in some .md? (not sure to which level of deatil we are describing configuration in .mds... to be checked)

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1 +1,2 @@
 - Using precise dependencies (~=) in packages.json
+- Provide default value (false) for tls.rejectUnauthorized config option
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Is this PR related with https://github.com/telefonicaid/perseo-fe/issues/272? To mention the issue number in that case in the CNR line...

Comment (cesarjorgemartinez) [reply CHANGES_NEXT_RELEASE]: It's a part for this issue, pending ignoreTLS/requireTLS

File: config.js (hunk 1 above)
Comment (AlvaroVega) [reply config.js]: Doc was updated.

**Pull request merged**